doc/release-notes/release-notes-bucash1.5.1.0.md

```diff
@@ -0,0 +1,193 @@
+`d63ceda7d` 'signdata' RPC call (#1458) (Andrew Stone)
+`dee3a88b6` Use SV upgrade specifications rather than SV 0.1.0 release notes (#1454) (Andrea Suisani)
+`2e0859704` Add instructions on how to activate SV upgrade (#1455) (Andrea Suisani)
+`4ccef23b0` depends: biplist 1.0.3 (fanquake)
+`786c1d18c` rsvg binaries are needed to convert svg to png while producing dmg on macos (#1453) (Andrea Suisani)
+`c5755e5ac` BUcash 1.5.0.1 release notes (#1452) (Andrea Suisani)
+`420e7d875` Bump BUcash version to 1.5.0.1 (Andrea Suisani)
+`94a9647f0` patch check that look at pointer address of labelPublic instead of size (Greg Griffith)
+`13d647e9b` Add lock on vNodes when checking size() in rpc getinfo() (Peter Tschipper)
+`b62f437b2` fix linker error (Andrew Stone)
+`b5984be52` Fix bug in dmg builder so that it actually reads in the configuration file (Don Patterson)
+`b9ded0885` build: Fix 'make deploy' for OSX (Cory Fields)
+`2178b1cfa` depends: mac deploy Py3 compatibility (Wladimir J. van der Laan)
+`b4ff18aec` Add RPC call 'logline' to add lines to debug.log (Awemany)
+`b3ef45ed9` Fix requestmanager_tests.cpp (Peter Tschipper)
+`7a84f29d2` Add Bitcoin SV seeders only if SV fork is activated (Andrea Suisani)
+`057295cc7` Turn miningSvForkTime and minigForkTime into CTweakRef (Andrea Suisani)
+`a85fe1650` fix format (Peter Tschipper)
+`3c4c5f166` Fix getminingcandidate (Peter Tschipper)
+`d63fee097` Reduce Transaction re-requests under heavy load (#1430) (Peter Tschipper)
+`472f47990` Some cleanup and also extend CTransactionRef into a few more areas (#1429) (Peter Tschipper)
+`7f16a286c` Fix ctor.py from failing (Peter Tschipper)
+`c64d44b74` clean up merge of new opcodes, add SV config and implement max instruction increase.  Add a config validator for the forks.  To do so, it was necessary to extend the CTweak to support validators like CTweakRef currently does.  Shorten QA test time (Andrew Stone)
+`a669fe85b` from SV: Added OP_INVERT functionality and tests (shaunOK)
+`724076254` from SV: Add OP_MUL functionality and tests (shaunOK)
+`7194b0b79` Merge from SV: Added LSHIFT and RSHIFT opcode functionality and test (shaunOK)
+`06abba7a1` Fix for #1416: can't compile without wallet unless bdb4.8 is installed (#1427) (Peter Tschipper)
+`760e6b271` Add getraworphanpool rpc (#1421) (Peter Tschipper)
+`ace8ee652` Turn graphene on by default (#1418) (Peter Tschipper)
+`83e3b7b7b` Remove run-bitcoind-for-test.sh.in (#1424) (awemany)
+`05688d3a7` Define NODE_CF service bit (#1422) (Chris Pacia)
+`e59a724d2` Fix BIP 135 voting guide URL (#1417) (Andrea Suisani)
+`41cb9be0e` build: python 3 compatibility (Wladimir J. van der Laan)
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release:
+
+- Andrea Suisani
+- Andrew Stone
+- Awemany
+- George Bissias
+- Chris Pacia
+- Dimitris Apostolou
+- Greg Griffith
+- Jonas Schnelli
+- Peter Tschipper
+- Søren Bredlund Caspersen
```

Comment:
jtoomim should be in the list for his path that reduces the preferred timer... even though your configurable patch overrides it.

Reply:
Agreed